docs/components/related-navigation.md

```diff
@@ -0,0 +1,271 @@
+# List out all past documentation/supporting material with regards to or realted to this component. It can include (1) past design documentation, (2) research findings, and (3) presentations.
+insights:
+  # To add additional insights duplicate the the fields below (adhering to the formating) but increase the count by one integer.
+  0:
+    # A description is REQUIRED in order for this information to render on the page.
+    date: November 2021
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'This research explored how users who advise others interact with and navigate GOV.UK content page.
+
+
+      Users were found to have varied expectations for what "related content" might mean.
+      The phrase "related content" means different things to users depending on their prior experience of the content and of other resources.
+
+
+      "I do not know if these are maybe ones that have been recently clicked on by someone"
+
+
+      "[related content] might be things that are referred to in that document that if you want more detail on, you click into those pages"
+
+
+      "you need to click on all of these [related links] anyway, even if you do not understand what they are... because obviously there could be information there that you are missing that could help you".'
+
+    title: 'GOV.UK Explore: page level nav (advisers) - Research round 5'
+    link: https://docs.google.com/presentation/d/1wsiH0OJPyS9DtxvUXri-tNkqFhU6N00xjdsWSAHK2Fw/edit#slide=id.g1006224b8f4_0_113
+    documentFormat: Google Slides
+  1:
+    # A description is REQUIRED in order for this information to render on the page.
+    date: February 2023
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'This report expains what related links are and how they are generated, both algorythmically and manually.'
+    title: Related links and how they are generated
+    link: https://docs.google.com/presentation/d/1-_u-00zColPLAwUw1iYTBWiTa1O2OdUAx1Jie_Pn9Qg/edit#slide=id.g10d42026b8_2_0
+    documentFormat: Google Slides
+  2:
+    # A description is REQUIRED in order for this information to render on the page.
+    date: April 2023
+    description:
+      # You MUST wrap this in single quotation marks (ie. ' '), since markdown can be used to enter this information. To create a heading, use three hashes (ie. ###).
+      'A details analysis showed:
+
+      
+      43% of all clicks are of the first related link - this shows that lots of users only click the first link in ‘Related content’
+
+
+      Most users do not click more than 3 links
+      
+
+      Some advisers (‘advise a thing’ users) struggle to grasp the distinction between related content and ‘also part of’ links'
```

Comment:
Just wondering why "advise a thing" + "also part of" are wrapped in single quotes?